builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -376,9 +380,15 @@ node() {
   }
 }
 
-def deleteAndRedeployService(environment_logical_id, envBucketKey){
+def deleteAndRedeployService(environment_logical_id, envBucketKey, accountDetails, config){
+  def s3bucketvalue
+  for (item in accountDetails.REGIONS) {
```

Comment:
Write a common method for getting the s3bucketvalue

Reply:
Updated with a common function to get the bucket Value.